### ISSUE TITLE
Deprecate Code42 label, no response from vendor about a static URL for download

### ISSUE THREAD

**Issue by Truezeldafan** [Installomator/Installomator#572]
Currently, these links below from the C42 label:

https://download.code42.com/installs/agent/latest-mac.dmg
https://download.code42.com/installs/agent/latest-mac-arm64.dmg

Are not downloading the recently released version, 10.0. Code42 Representatives in the macadmins slack channel have stated the paths have been changed for the 10.0 release and there is currently not an updated "latest" link yet. They hope to provide one sometime.

**Comment by naschenbrenner** [comment]
It would appear both of the "legacy" URL's are offline so this workflow is no longer working. Has there been any update on new "latest" URL's from Code42? 


![SCR-20221114-f76](https://user-images.githubusercontent.com/60478794/201719905-8a0d4452-283d-441e-81c3-70bace2cbf20.png)

**Comment by acodega** [comment]
I think we're going to have to remove the label.

**Comment by rchase254** [comment]
I tested the script locally using the following URL: https://download.crashplan.com/installs/agent/latest-mac.dmg. This link downloaded version 10.4.  Replacing it in the Installomator script successfully downloaded and installed the application.

**Comment by naschenbrenner** [comment]
crashplan split off from code42 a few months back, they're separate products now so that link unfortunately doesn't point to the right installer called out in this issue.

**Comment by orrison-CrashPlan** [comment]
Hi all, I'm an employee over at CrashPlan weighing in to try and help disambiguate things just a tad. 

We at CrashPlan did split off from Code42 a few months ago. We also just released our version 11 app which is branded CrashPlan and so should make things a bit clearer moving forward. At this point we should have three different labels for CrashPlan/Code42. I can't fully comment on the Code42 piece since I am not employed by Code42.

1. A Code42 label, the one currently in place I suspect will work great. 
https://github.com/Installomator/Installomator/blob/main/fragments/labels/code42.sh

2. A CrashPlan label. Built like this:

```
crashplan)
    name="CrashPlan"
    type="pkgInDmg"
    downloadURL="https://download.crashplan.com/installs/agent/latest-mac.dmg"
    expectedTeamID="UGHXR79U6M"
    blockingProcesses=( NONE )
    ;;
```
3. A CrashPlan small business label as discussed in this pull request. That will have a similar download URL, and team ID as the main CrashPlan one.
https://github.com/Installomator/Installomator/pull/931

**Comment by lennyvaknine43** [comment]
None of the URLs are pointing to v12 of the installer which is what we require since they separated the AAT from the backup agent.

**Comment by acodega** [comment]
Any Code42 users know if this label can be updated for version 12 and beyond? Otherwise we may need to remove the label in a future release.

**Comment by naschenbrenner** [comment]
It looks like they're still using a dynamic URL per version, I think at this point it's best the label is removed and if this ever changes myself or someone can put in a PR to add it back.

**Comment by acodega** [comment]
Changing Issue topic now to depreceate.

**Comment by acodega** [comment]
Fixed in #766 (Excuse the lateness, doing some housekeeping on Issues)
